plugins/language_json.lua

```diff
@@ -0,0 +1,21 @@
+-- mod-version:3 priority:90
```

Comment:
`110` would probably work better I think.

Reply:
Yeah, that works